### ISSUE TITLE
Specify order of industries in "by industry" output table

### ISSUE THREAD

**Issue by jdebacker** [OpenSourcePolicyCenter/webapp-public#385]
Currently,  assets and industries are listed in alphabetical order.  Industries should be ordered as:

```
All Industries
  Agriculture, forestry, fishing, and hunting
   Farms
   Forestry, fishing, and related activities

  Mining
   Oil and gas extraction
   Mining, except oil and gas
   Support activities for mining

  Utilities

  Construction                                 

  Manufacturing
   Durable goods
      Wood products
      Nonmetallic mineral products
      Primary metals
      Fabricated metal products
      Machinery
      Computer and electronic products
      Electrical equipment, appliances, and components
      Motor vehicles, bodies and trailers, and parts
      Other transportation equipment
      Furniture and related products
      Miscellaneous manufacturing
   Nondurable goods
      Food, beverage, and tobacco products
      Textile mills and textile product mills
      Apparel and leather and allied products
      Paper products
      Printing and related support activities
      Petroleum and coal products
      Chemical products
      Plastics and rubber products

  Wholesale trade

  Retail trade

  Transportation and warehousing
   Air transportation
   Railroad transportation
   Water transportation
   Truck transportation
   Transit and ground passenger transportation
   Pipeline transportation
   Other transportation and support activities 
   Warehousing and storage

  Information
   Publishing industries (including software)
   Motion picture and sound recording industries
   Broadcasting and telecommunications
   Information and data processing services

  Finance and insurance
   Federal Reserve banks
   Credit intermediation and related activities
   Securities, commodity contracts, and investments
   Insurance carriers and related activities
   Funds, trusts, and other financial vehicles

  Real estate and rental and leasing
   Real estate
   Rental and leasing services and lessors of intangible assets

  Professional, scientific, and technical services
   Legal services
   Computer systems design and related services
   Miscellaneous professional, scientific, and technical services

  Management of companies and enterprises

  Administrative and waste management services
   Administrative and support services
   Waste management and remediation services

  Educational services                         

  Health care and social assistance
   Ambulatory health care services
   Hospitals
   Nursing and residential care facilities
   Social assistance                    

  Arts, entertainment, and recreation
   Performing arts, spectator sports, museums, and related activities
   Amusements, gambling, and recreation industries

  Accommodation and food services
   Accommodation
   Food services and drinking places

  Other services, except government
```

**Comment by talumbau** [comment]
@PeterDSteinberg switching this over to you as we discussed.

**Comment by PeterDSteinberg** [comment]
@jdebacker I am also getting a KeyError when trying to refer to `Federal Reserve banks` - doesn't look like it's in the dataframes.  This is the only one I am missing now.

**Comment by jdebacker** [comment]
@PeterDSteinberg - please excuse me again - and drop that from the list of industries to be listed in the output table.  The Fed banks don't pay taxes so, we don't need to report them here.  Thanks!

**Comment by PeterDSteinberg** [comment]
Fixed by #392
